src/data/nedb/db.js

```diff
@@ -1,48 +1,47 @@
-var _ = require( "lodash" );
-var path = require( "path" );
-var nedb = require( "nedb" );
-var when = require( "when" );
-var nodeWhen = require( "when/node" );
-var Datastore = require( "nedb" );
-var config = require( "configya" )( {
+const _ = require( "lodash" );
+const path = require( "path" );
+const when = require( "when" );
```

Comment:
I left when in here because this file also uses when/node. I checked the bluebird api and leaving when/node in here seemed like it might be the most suitable option.